docs/src/architecture/accounts.md

```diff
@@ -52,6 +52,27 @@ Interface for accounts. In Miden every account is a smart contract. It has an in
 
 *Note: Since code in Miden is expresed as MAST, every function is a commitment to the underlying code. The code cannot change unnoticed to the user because its hash would change. Behind any MAST root there can only be `256` functions*
 
+#### Example Account Code
+Currently, Miden provides two standard implementations for account code. 
+
+A basic user account exposes the following code as its interface.
+
+```
+    use.miden::contracts::wallets::basic->basic_wallet
+    use.miden::contracts::auth::basic
+
+    export.basic_wallet::receive_asset
+    export.basic_wallet::send_asset
+    export.basic::auth_tx_rpo_falcon512
+
+```
+
+That means, [note scripts](https://0xpolygonmiden.github.io/miden-base/architecture/notes.html#note-scripts) or transaction scripts can call those functions and in doing so, the account can receive and send assets and authenticate the transaction. It is important to know, that without correct authentication, a note cannot successfully invoke receive and send asset.
```

Comment:
It was not clear for me at first that it was not other users with their accounts that we calling functions on other users accounts. 

But that note scripts and tx scripts were calling functions on the api of the note consumer. Idk if it would be suited to add this here, but would be great to have this explanation somewhere.

Reply:
This is super valuable feedback. I am already to blind to see this. Nice